modules/op-pipelines-concepts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id='op-pipelines-concepts_{context}']
+= {pipelines-title} concepts
+
+{pipelines-title} provide the following set of standard Custom Resource Definitions (CRDs) that act as the building blocks from which you can assemble a CI/CD pipeline for your application:
+
+Task:: A Task is the smallest configurable unit in a Pipeline. It is essentially a function of inputs and outputs that form the Pipeline build. It can run individually or as a part of a Pipeline. A Pipeline includes one or more Tasks, where each Task consists of one or more steps. Steps are a series of commands that are sequentially executed by the Task.
+
+Pipeline:: A Pipeline consists of a series of Tasks that are executed to construct complex workflows that automate the build, deployment, and delivery of applications. It is a collection of PipelineResources, Parameters, and one or more Tasks. A Pipeline interacts with the outside world by using PipelineResources, which are added to Tasks as inputs and outputs.
```

Comment:
Resources is capitalized inconsistently in this paragraph. Is it intentional based on the objects?

Reply:
Yes, PipelineResources is a CRD defined in OpenShift Pipelines. Since in this section we are introducing the concepts definitions, we will be using the objects/components names defined here as is throughout the document.